### ISSUE TITLE
Fil Manager: Save Changes button

### ISSUE THREAD

**Issue by RosieRoylance-White** [mothership-ec/cog-mothership-cp#159]
Another button disappearing, since the last push. Cannot save Alt text and tags, have to exit into file manager and then click back into the image to save.

![screen shot 2013-09-24 at 17 29 26](https://f.cloud.github.com/assets/4669476/1201550/a175ddd4-2536-11e3-89ed-277f88487b98.png)
